lib/ex_doc/formatter/html/templates/footer_template.eex

```diff
@@ -7,7 +7,15 @@
           <span class="line">
             designed by
             <a href="https://twitter.com/dignifiedquire" target="_blank" rel="noopener" title="@dignifiedquire">Friedel Ziegelmayer</a>.
-            </span>
+          </span>
+        </p>
+        <p>
+          <button class="line footer-button night-mode-toggle">
```

Comment:
The "toggle night mode" is not working for me on the latest demo (autocomplete_mod_fun).

Reply:
Fixed!